po/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# pushing/pulling translations to zanata
+
+file (RELATIVE_PATH SRCDIR ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_SOURCE_DIR}/libdnf)
+ADD_CUSTOM_TARGET (gettext-export
+                   find ${SRCDIR} -iname "*.c" |
+                   xargs xgettext --from-code=UTF-8 --keyword=_ --keyword=N_ --keyword=C_:1c,2 --keyword=NC_:1c,2 --keyword=g_dgettext:2 --keyword=g_dcgettext:2 --keyword=g_dngettext:2,3 --keyword=g_dpgettext2:2c,3 -c --output=libdnf.pot &&
```

Comment:
why do you specify all keywords manually? why not `-L C`?

Reply:
> why do you specify all keywords manually?
Is not true. I do not specify all keywords manually. I specify only additional keywords which are added by glib. 
Because, according xgettext documentation only this keywords are defined:
For C, C++, and GCC-source: gettext, dgettext:2, dcgettext:2, ngettext:1,2, dngettext:2,3, dcngettext:2,3, gettext_noop, and pgettext:1c,2, dpgettext:2c,3, dcpgettext:2c,3, npgettext:1c,2,3, dnpgettext:2c,3,4, dcnpgettext:2c,3,4. 

> why not -L C ?
Language is automatically detected from file name suffix.